### PR TITLE
Viewer crashing after 10 points plotting issue resolved

### PR DESCRIPTION
**Pull request by ruchimotwaniBC**
This PR is for issue  https://github.com/dcs4cop/xcube-viewer/issues/299

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -238,7 +238,7 @@ const userPlaceColorsArray: [string, Color][] = [
     ["purple", purple],
     ["indigo", indigo],
     ["cyan", cyan],
-    ["deepPurple", deepPurple],
+    ["purple", purple],
```
Comment (AliceBalfanz) [inline src/config.ts:241]: Please note, that purple is already used in the list. Please choose a different color from this list https://www.w3.org/wiki/CSS/Properties/color/keywords which is not used yet.

**Review by AliceBalfanz — CHANGES_REQUESTED** [review]
For the PR, please also add a changlog entry:  https://github.com/dcs4cop/xcube-viewer/blob/master/CHANGES.md

**Review by AliceBalfanz — APPROVED** [review]
Thanks for adjusting the PR according to the comments and for fixing the issue :)

File: src/config.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 import { Color, PaletteMode } from '@mui/material';
 import {
+    brown,
     cyan,
     deepPurple,
```
Comment (pont-us) [inline src/config.ts:29]:
```suggestion
    brown,
    cyan,
```

If we're not using deepPurple any more, we can presumably remove the import.

File: src/config.ts [hunk 3]
```diff
@@ -238,7 +239,7 @@ const userPlaceColorsArray: [string, Color][] = [
     ["purple", purple],
     ["indigo", indigo],
     ["cyan", cyan],
-    ["deepPurple", deepPurple],
+    ["brown",brown]
```
Comment (pont-us) [inline src/config.ts:242]:
```suggestion
    ["brown", brown]
```

**Review by pont-us — APPROVED** [review]
Approved, subject to the suggested changes.

I'm actually a little confused about my initial diagnosis of the bug's cause now, since I see that deepPurple is actually defined in @mui/material/colors/deepPurple.js. But if the deepPurple -> brown substitution is tested and working then that's good enough for now -- perhaps the colour name is being used somewhere rather than the defined value in the look-up table? I don't know the viewer code well enough to be sure.

**Pull request merged**